docs/develop/dotnet/message-passing.mdx

```diff
@@ -0,0 +1,109 @@
+---
+id: message-passing
+title: Application message passing - .NET SDK feature guide
+sidebar_label: Messages
+description: Applications can send and receive messages using Signals, Queries, and Updates.
+toc_max_heading_level: 5
+keywords:
+  - message passing
+  - signals
+  - queries
+  - updates
+tags:
+  - message-passing
+  - signals
+  - queries
+  - updates
+---
+
+<!-- What is on this page-->
```

Comment:
Inconsistent with whether you provide this or expect it to be provided. I would expect this could have already been filled in the way it is in some other pages.